app/routes/matador.tsx

```diff
@@ -3,10 +3,10 @@ import { ColorSchemeProvider, MantineProvider } from "@mantine/core";
 import type { LinksFunction, LoaderFunction } from "@remix-run/node";
 import { Outlet, useCatch, useLoaderData } from "@remix-run/react";
 import { useEffect, useState } from "react";
-import { ErrorFallback, NavBar } from "~/lib/matador/components/index";
-import { themeKeyLocalStorage } from "~/lib/matador/constants";
+import { NavBar } from "~/lib/matador/components";
+import { ErrorFallback } from "~/lib/matador/components/errors";
```

Comment:
Why import ErrorFallback from `~/lib/matador/components/errors` and not from `~/lib/matador/components`?